### ISSUE TITLE
Link with application web plugin : bug ?

### ISSUE THREAD

**Issue by EC2311** [ericferon/glpi-archisw#30]
Hello

In an web application plugin, we can mak a link to structured application either with "attached item" (where we can see 2 items) either "structured application" where nothing appears.
![image](https://user-images.githubusercontent.com/42341169/58477408-bd514780-8153-11e9-8fb6-550c1b1b1260.png)

but as you can see, web application is linked to structured application 
![image](https://user-images.githubusercontent.com/42341169/58477479-ebcf2280-8153-11e9-88e7-ba7612360d4b.png)

but nothing appear in the 'structured application (second choice, in the 1st picture) : why ?
What is the difference between twice ?

**Comment by ericferon** [comment]
For each "class" of items (f.i archisw, webapplication, database, dataflow, ...), there is a table (in the GLPI database) with associated elements. You can see all the associated elements, whatever their type, in the tab "Associated Items" (f.i all "webapplications" associated with an "archisw" item).
The other way round, you can see in the detailed form of an associated item (f.i a webapplication), all the tabs of items to which it has been associated (f.i the "archisw" item).
Of course, you can do the same in the "webapplication" (i.e link to "archisw"), but in this case the link is stored in the "webapplication" table.
In summary, you have to choose which is your master, where you want to see mainly the links.
Best regards,
